docs/Makefile

```diff
@@ -29,8 +29,10 @@ help:
 
 clean:
 	rm -rf $(BUILDDIR)/*
+	rm -rf ./source/notebooks
 
 html:
+	rm -rf ./source/notebooks/ && mkdir ./source/notebooks && cp -r ./../share/jupyter/notebook_templates/ipywidgets/*  ./source/notebooks/
```

Comment:
I am not very happy that we need to refer to the current (makefile) directory as `.`. There is no variable storing the location of the makefile.